packages/lit-html/package.json

```diff
@@ -155,6 +152,129 @@
     "!/development/test/",
     "/directives/"
   ],
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
```

Comment:
Should this include "build:ts"? It's transitively included, but it seems better to be explicit about it

Reply:
Done.